moped-editor/package.json

```diff
@@ -71,27 +71,28 @@
     "filepond-plugin-file-validate-size": "^2.2.8",
     "filepond-plugin-image-exif-orientation": "^1.0.11",
     "filepond-plugin-image-preview": "^3.1.6",
-    "formik": "^2.4.3",
+    "formik": "^2.4.5",
     "graphql": "^15.8.0",
     "history": "^5.3.0",
     "html-react-parser": "^1.4.14",
     "lodash.foreach": "^4.5.0",
     "lodash.get": "^4.4.2",
     "lodash.isequal": "^4.5.0",
     "lodash.toarray": "^4.4.0",
+    "mapbox-gl": "^3.0.1",
```

Comment:
This used to get implied into the dependencies through another package, but needs to be explicitly installed now.